tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java

```diff
@@ -127,6 +156,11 @@ private TestInfo() {
       mSize = ThreadLocalRandom.current().nextInt(0, MAX_SIZE);
     }
 
+    private TestInfo(int size) {
```

Comment:
This constructor should just take both id and size, and the other constructor should just call this one.